crates/task-impls/src/consensus.rs

```diff
@@ -775,19 +775,20 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, A: ConsensusApi<TYPES, I> +
                         id: self.id,
                         registry: self.registry.clone(),
                     };
-                    let name = "Quorum Vote Collection";
-                    self.vote_collector = spawn_vote_accumulator::<
+                    *collector = create_vote_accumulator::<
                         TYPES,
                         QuorumVote<TYPES>,
                         QuorumCertificate<TYPES>,
-                    >(
-                        &info, vote.clone(), event, name.to_string()
-                    )
+                    >(&info, vote.clone(), event)
                     .await;
-                } else if let Some((_, _, stream_id)) = self.vote_collector {
-                    self.event_stream
-                        .direct_message(stream_id, HotShotEvent::QuorumVoteRecv(vote.clone()))
-                        .await;
```

Comment:
Just wanted to make sure I understand correctly why we remove `direct_message` here. Looks like we are no longer sending `*VoteRecv` in the associated task implementations. Is it because we've already had the network sending `Recv` events for all `Send` events?

It's a bit weird since I used to think it made sense to also have these tasks send `Recv` events again, but I'm not sure now, so wanted to double-check.

Reply:
Before this PR both the network task,  these direct message calls would send the `Recv` events.  If just removed `direct_send` then the task would miss some events.  Every event that came in before the task is created would not be seen and it caused some bugs before.  We can't get rid of the network event sending these events because then nobody would receive them